### PR TITLE
fix: Compile template at startup

### PR DESCRIPTION
**Pull request by alicebob**
Fail as fast as possible.

(yup, I managed to break the template...)

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [x] None of the above
* [x] I have read the **CONTRIBUTING** document.
* [x] `make ci` passes on my machine.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: pipeline/release/body.go [hunk 1]
```diff
@@ -23,6 +23,12 @@ const bodyTemplate = `{{ .ReleaseNotes }}
 Automated with [GoReleaser](https://github.com/goreleaser)
 Built with {{ .GoVersion }}`
 
+var bodyTemplateT *template.Template
+
+func init() {
+	bodyTemplateT = template.Must(template.New("release").Parse(bodyTemplate))
```
Comment (caarlos0) [inline pipeline/release/body.go]: maybe we could change the `bodyTemplate` var to `bodyTemplateText` and this one to `bodyTemplate` only... what do you think?

**Comment by caarlos0** [comment]
Thanks!!

Just a very small comment :)

**Comment by alicebob** [comment]
Sure :)

The original was the smallest thing possible. I push-f-ed the amended commit.

**Comment by caarlos0** [comment]
thanks @alicebob :D

**Pull request merged**